docs/content/product-vocabulary.stories.mdx

```diff
@@ -55,6 +59,26 @@ leads.
 
 <br />
 
+### Jobber / Jobber.com / Jobber mobile app
+
+Refer to our overall software regardless of platform as **Jobber**.
```

Comment:
Is it overkill to include background info here? (For ex. "To differentiate platform, describe to the user how the experience is accessed - Jobber can be accessed through the Jobber mobile app, or on any device by logging in at jobber.com.")

Reply:
Adjusted the second line a bit since the examples go into mobile app vs jobber.com. Lemme know if you think that works!

Reply:
Beauty! :chefs-kiss:

Reply:
You have the power to approve this (from the files changed tab I believe) if you feel comfortable doing so!

Reply:
Do I? ...... :blob-sweats: